### PR TITLE
DB-9079 Support CURRENT SERVER

### PR DESCRIPTION
**Pull request by arnaud-splice**

### CODE REVIEW

**Comment by arnaud-splice** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java [hunk 1]
```diff
@@ -99,149 +99,164 @@ public void testSinFunction() throws Exception{
...
-				ps.setString(1, data[i][0]);
-				ps.setString(2, data[i][1]);
-				ps.setString(3, data[i][2]);
-				ResultSet rs = ps.executeQuery();
-				Assert.assertTrue(rs.next());
-				Assert.assertEquals("Values do not match",rs.getString(1),data[i][3]);
-				Assert.assertFalse(rs.next());
-			}
-	}
+        ResultSet rs = methodWatcher.executeQuery(sqlText);
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java]: You can use "try-with-resources" to ensure resultsets get closed:
try(ResultSet rs = methodWatcher.executeQuery(sqlText)) {
assertEquals("\n" + sqlText + "\n", expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
}

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java [hunk 2]
```diff
@@ -99,149 +99,164 @@ public void testSinFunction() throws Exception{
...
+        assertEquals("\n" + sqlText + "\n", expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
+        rs.close();
 
+        rs = methodWatcher.executeQuery(sqlTextAlt);
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java]: close this rs

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java (hunk 1 above)
Comment (arnaud-splice) [reply splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java]: Done

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java (hunk 2 above)
Comment (arnaud-splice) [reply splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java]: Done

**Pull request merged**